NitroxClient/GameLogic/LocalPlayer.cs

```diff
@@ -35,7 +42,7 @@ public LocalPlayer(IMultiplayerSession multiplayerSession, IPacketSender packetS
 
         public void BroadcastStats(float oxygen, float maxOxygen, float health, float food, float water)
         {
-            PlayerStats playerStats = new PlayerStats(multiplayerSession.Reservation.PlayerId, oxygen, maxOxygen, health, food, water);
+            PlayerStats playerStats = new PlayerStats(PlayerContext, oxygen, maxOxygen, health, food, water);
```

Comment:
I'm not sure that the correct solution is to add player context to all of these packets.  That is going to add a lot of unneeded data transfer.  I believe when a player connects (or we receive the initial sync packet) we can locally store the context for later use.